components-splunk/src/main/java/org/talend/components/splunk/connection/TSplunkEventCollectorConnection.java

```
@@ -0,0 +1,78 @@
+// ============================================================================
+//
+// Copyright (C) 2006-2016 Talend Inc. - www.talend.com
+//
+// This source code is available under agreement available at
+// %InstallDIR%\features\org.talend.rcp.branding.%PRODUCTNAME%\%PRODUCTNAME%license.txt
+//
+// You should have received a copy of the agreement
+// along with this program; if not, write to Talend SA
+// 9 rue Pages 92150 Suresnes, France
+//
+// ============================================================================
+package org.talend.components.splunk.connection;
+
+import java.io.IOException;
+
+import org.apache.http.HttpHost;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.DefaultHttpClient;
+
+/**
+ * created by dmytro.chmyga on Apr 26, 2016
+ */
+public class TSplunkEventCollectorConnection {
+
+    private HttpClient client = null;
+    
+    /**
+     * create http client to be used for connection with Splunk server
+     */
+    public void connect() {
+        if(client == null) {
+            try {
+                client = new DefaultHttpClient();
+            } catch (Exception e) { 
```

It is no good to leave empty catch clause
add logger.debug(e.getMessage()); maybe